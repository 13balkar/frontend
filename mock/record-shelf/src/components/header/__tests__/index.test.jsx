import { render } from '@testing-library/react';
import * as React from 'react';
import Header from '..';

jest.mock('react-router-dom', () => ({
  useNavigate: () => jest.fn()
}));

describe('Header', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<Header />);
    expect(asFragment()).toMatchSnapshot();
  });
});
