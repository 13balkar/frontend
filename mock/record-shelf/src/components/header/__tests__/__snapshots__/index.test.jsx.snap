// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render correctly 1`] = `
<DocumentFragment>
  <div
    class="header"
  >
    <p>
      My 
      <b>
        Record
      </b>
       Shelf
    </p>
  </div>
</DocumentFragment>
`;
